src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -30,11 +30,31 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.Display.ByGeometryColor</oldName>
-    <newName>Display.Display.ByGeometryColor</newName>
+    <newName>Modifiers.GeometryColor.ByGeometryColor</newName>
```

Comment:
@Randy-Ma I would reconsider renaming namespaces and suggest changing only class names. We need to keep in mind that renaming namespaces also affects namespace mapping in Thunderstorm @gregmarr. Please refer to @ikeough's comments on the #dynamo-design-script slack channel: https://autodesk.slack.com/archives/C187950LA/p1491917642623367